modules/auxiliary/scanner/ftp/ftp_version.rb

```diff
@@ -18,31 +18,24 @@ def initialize
       'Author'      => 'hdm',
       'License'     => MSF_LICENSE
     )
-
-    register_options(
-      [
-        Opt::RPORT(21),
-      ], self.class)
   end
 
-  def run_host(target_host)
+  def peer
+    "#{rhost}:#{rport}"
+  end
 
+  def run_host(_target_host)
     begin
-
-    res = connect(true, false)
-
-    if(banner)
-      banner_sanitized = Rex::Text.to_hex_ascii(self.banner.to_s)
-      print_status("#{rhost}:#{rport} FTP Banner: '#{banner_sanitized}'")
-      report_service(:host => rhost, :port => rport, :name => "ftp", :info => banner_sanitized)
-    end
-
-    disconnect
-
+      if connect(true, false) && banner && banner =~ /^220[ -]/
```

Comment:
The common prefix `Regexp`, `/^220[ -]/`, can be put in a constant and used in these two places so later developer know the common prefix is intentional and not coincidental.